app/models/ability.rb

```diff
@@ -4,27 +4,28 @@ class Ability
   def initialize(user)
     if user
       can :manage, User, id: user.id
-      can :manage, GroupedIssue do |group|
-        user.is_owner_of?(group.website)
-      end
+      can :manage, GroupedIssue
       can :manage, Issue do |issue|
-        user.is_owner_of?(issue.group.website)
+        user.is_owner_of?(issue.website)
       end
       can :manage, Message do |message|
-        user.is_owner_of?(message.issue.group.website)
+        user.is_owner_of?(message.subscriber.website)
       end
```

Comment:
if we change :manage, GroupedIssue line then the invited user is not able to access /error#show
I tried `[:read, :update], GroupedIssue` too